app/src/renderer/css/main.css

```diff
@@ -15,6 +15,15 @@
 @import 'components/_buttons';
 @import 'components/_progress';
 
+/* Body */
+body {
+  transition: all .2s ease;
+}
+
+body.is-tray-arrow-hidden {
+  padding-top: 10px;
```

Comment:
Need to animate padding on body, otherwise the window slides up when tray arrow is hiding.